pxr/imaging/plugin/hioOiio/oiioImage.cpp

```diff
@@ -289,7 +289,7 @@ _FindAttribute(ImageSpec const & spec, std::string const & metadataKey)
     bool convertMatrixTypes = false;
     std::string key = _TranslateMetadataKey(metadataKey, &convertMatrixTypes);
 
-    ImageIOParameter const * param = spec.find_attribute(key);
+    ParamValue const * param = spec.find_attribute(key);
```

Comment:
I did not add any preprocessor checking here, since it looks like the `ImageIOParameter` alias was in place (and already deprecated) at least as far back as OpenImageIO 2.0:
https://github.com/AcademySoftwareFoundation/OpenImageIO/blob/88df1f55795f79db0c702892fcb311a16509dc04/src/include/OpenImageIO/imageio.h#L106

I'm not sure if anyone is still building against a 1.x version of OpenImageIO, but I can try to distill an `OIIO_VERSION` to use as a guard here if that's needed.